### PR TITLE
Remove Internet Explorer support

### PR DESCRIPTION
**Pull request by PVince81**
Removes IE detection and workarounds.
Also removes public symbols for IE detection.

This might break apps that are relying on the USER_AGENT_IE constant.

### CODE REVIEW

**Comment by skjnldsv** [comment]
Might fail hard :thinking: 

Also see https://github.com/nextcloud/server/pull/26358

**Comment by PVince81** [comment]
hmm, okay, so that PR removes even more on the JS side.
then maybe that PR can be merged first

File: lib/private/AppFramework/Http/Request.php [hunk 1]
```diff
@@ -63,7 +63,6 @@
  * @property mixed[] server
  */
 class Request implements \ArrayAccess, \Countable, IRequest {
-	public const USER_AGENT_IE = '/(MSIE)|(Trident)/';
```
Comment (nickvergessen) [inline lib/private/AppFramework/Http/Request.php:66]: lets keep that?

**Comment by skjnldsv** [comment]
Closing?

**Comment by PVince81** [comment]
yeah, closing in favor of https://github.com/nextcloud/server/pull/26358

**Pull request closed without merging**